actions/command.ts

```diff
@@ -60,6 +61,18 @@ export function executeCommand(message: string, args: CommandArgs): ActionFunc {
             msg = cmd + ' ' + msg.substring(cmdLength, msg.length).trim();
         }
 
+        // Add track event for certain slash commands
+        const commandsWithTelemetry = [
```

Comment:
just below there is a switch for the command, why not sending the trackEvent in each particular case?
`case '/marketplace':` ... case '/help':

Reply:
Not all commands go through the switch, only the ones that have a direct frontend action. `/help` for example does not have a case.